tests/unit/LHNFilterTest.js

```diff
@@ -247,17 +247,17 @@ describe('Sidebar', () => {
 
             // Given these combinations of booleans which result in the report being filtered out (not shown).
             const booleansWhichRemovesInactiveReport = [
-                // isUserCreatedPolicyRoom
-                JSON.stringify([false, false, true, false, false, false, false]),
+                // hasComments is false
+                JSON.stringify([false, false, false, false, false, false, false]),
 
-                // isUserCreatedPolicyRoom, isUnread
```

Comment:
isUserCreatedPolicyRoom should always show, regardless of if there's no message if they aren't archived, so this fixes that